features/command_line_interface/smells_count.feature

```diff
@@ -39,7 +41,8 @@ Feature: Reports total number of code smells
     """
 
   Scenario: Output total number of smells even if total equals 0
-    When I run reek spec/samples/three_clean_files
+    Given a directory with clean files
```

Comment:
I love this cleanup but i would still prefer to mention the file names explicitly since this would be clearer IMHO, what do you think?